frontend/src/metabase/containers/DataPicker/tests/DataPicker-Common.unit.spec.ts

```diff
@@ -49,10 +48,12 @@ describe("DataPicker", () => {
       filters: { types: type => type !== "questions" },
     });
 
-    expect(await screen.findByText(/Orders/i)).toBeInTheDocument();
-    expect(screen.getByText(/Sample Database/i)).toBeInTheDocument();
-    SAMPLE_DATABASE.tables.forEach(table => {
-      expect(screen.getByText(table.displayName())).toBeInTheDocument();
+    expect(
+      await screen.findByText(SAMPLE_TABLE.display_name),
+    ).toBeInTheDocument();
+    expect(screen.getByText(SAMPLE_DATABASE.name)).toBeInTheDocument();
+    SAMPLE_DATABASE.tables?.forEach(table => {
```

Comment:
I know it's because of how we're creating our types. But this looks a bit weird like we're saying Oh in this test, the `tables` could be null, so it's possible `expect` at line 56 isn't called at all.

I don't know if there's a way to create a new type that has the properties required when we deliberately pass that property to a factory function. Because you literally pass `tables` [here](https://github.com/metabase/metabase/pull/27761/files#diff-8b3e47012def8ac681e8617debf3840fe3bab94690ff7addba69c22140a51e6bR57), but the type still coming out as if `tables` could be null